README.md

```diff
@@ -321,10 +321,7 @@ public class MainFragment extends Fragment implements ConnectionStatusListener {
             return;
         }
         
-        // Login and create account 
-        // a) Static token variant:
-        dataCapturingService.registerJWTAuthToken(username, token);
```

Comment:
because this is in the "CyfaceDCS only" section so there should not be any mentions of the MovebisDCS code usage